docs/analysis-workflows/dna-open-access-filtering.md

```diff
@@ -1,8 +1,47 @@
 ---
 id: dna-open-access-filtering
-title: Open Access Filtering
-sidebar_label: Open Access Filtering
+title: Open Access Variant Filtering
+sidebar_label: Open Access Variant Filtering
 platform_key: DOCS_DNA_PIPELINE
 ---
 
-![Reminder Banner](/assets/submission/banner-reminder.svg)
+The SNV/InDel variant calls from both Sanger & MuTect2 calling results on ARGO Data Platform are further processed through the Open Access Variant Filtering Workflow to create VCF files that can be downloaded anonymously from the File Repository. For details about the process workflow, please see the latest version of the [ARGO Open Access Variant Filtering Workflow](https://github.com/icgc-argo-workflows/open-access-variant-filtering/releases).
+
+## Background
+
+In cancer research it is not thought to be possible to re-identify patients from somatic variants data, as these mutations are found in the tumour not in an individual's germline genome. That means somatic variants do not provide identifying information theoretically and barrier-free release of somatic variants data can occur without compromising patient privacy.
+
+However, germline variants can be mistakenly identified as somatic variants, these are referred to as "gerline leaks" because of:
+
+- imperfect variant calling tools,
+- sequencing errors,
+- low and uneven reads coverage.
+
+A recent study by [Sendorek, D.H. et al. 2018](https://doi.org/10.1186/s12859-018-2046-0) indicated that both germline leaks in somatic SNVs, and the overlap rate between somatic SNVs with the germline database, are relatively low. Another study by [Meyerson, W. et al.](https://doi.org/10.1186/s12859-020-3508-8) also inferred that among the shared variants between somatic SNVs with the germline database, only 4% are potential germline leaks, while the other variants mostly represent true biological variants that just arose independently in both germline and somatic settings.
+
+It was reported by [Elena Rojano, et al.](https://doi.org/10.1093/bib/bby039) that more than 88% of disease-associated variants are in non-coding regions especially in regulatory regions, e.g:
+
+- cis and trans-regulatory elements such as promoters, enhancers,
+- transcribed non-coding regions with regulatory roles, such as miRNAs and lncRNAs.
+
+We want to make as many somatic variants as possible to be public accessible to the research community, with minimum risk of potential germline leaks. Therefore, we have taken the various genomic elements according to latest version of GENCODE annotations (e.g., v38), and have defined the open access regions to include the genomic regions of highest interest. For details about the definition of open access genomic regions, please refer to [Open-access Regions](https://github.com/icgc-argo/open-access-regions).
```

Comment:
- [ ] change "to be public accessible" to "publicly accessible"
- [ ] change "according to latest version of GENCODE annotations" to "according to the latest version of GENCODE annotations"